test/jdk/javax/swing/JInternalFrame/6725409/bug6725409.java

```diff
@@ -146,20 +139,11 @@ private void checkResult() {
 
     private static void sync() {
         robot.waitForIdle();
-    }
-    private static ExtendedRobot createRobot() {
-        try {
-             ExtendedRobot robot = new ExtendedRobot();
-             return robot;
-         }catch(Exception ex) {
-             ex.printStackTrace();
-             throw new Error("Unexpected Failure");
-         }
+        robot.delay(500);
```

Comment:
I still think `robot.delay(500);` is unnecessary in this test because the test never accesses UI.

By adding `delay(500)` to the `sync` method you make the test run longer… without a reason.